package.json

```diff
@@ -11,14 +11,17 @@
   "scripts": {
     "start": "nodemon server.js",
     "lint": "eslint .",
-    "lint:fix": "eslint . --fix"
+    "lint:fix": "eslint . --fix",
+    "test": "jest --coverage"
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@azure/identity": "^1.0.3",
+    "@azure/keyvault-secrets": "^4.0.4",
     "@hapi/joi": "^17.1.1",
     "apollo-server": "^2.14.5",
-    "bcrypt": "^5.0.0",
+    "azure": "^2.3.1-preview",
```

Comment:
Can you install azure package without preview suffix?

Reply:
I